test/e2e/external/dataplane/connectivity.go

```diff
@@ -0,0 +1,114 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dataplane
+
+import (
+	"fmt"
+	"sort"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+)
+
+const (
+	extAppName = "ext-app"
+	extNetName = "pseudo-ext"
+
+	testImage         = "registry.access.redhat.com/ubi7/ubi:latest"
+	testContainerName = "ext-test-container"
+)
+
+var (
+	simpleHTTPServerCommand = []string{"python", "-m", "SimpleHTTPServer", "80"}
+)
+
+var _ = Describe("[external_dataplane] Connectivity", func() {
```

Comment:
nit but in similar places we use '-' instead of '_'.

```suggestion
var _ = Describe("[external-dataplane] Connectivity", func() {
```